### PR TITLE
Activate "/etc/nagios/private" as private resource location

### PR DESCRIPTION
**Pull request by ghost**
RPM based installations use /etc/nagios/private as for resource.cfg: manage matching directory for RHEL and fedora.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ nagios Cookbook CHANGELOG
 This file is used to list changes made in each version of the nagios cookbook.
 
 
+v5.3.5
```
Comment (tas50) [inline CHANGELOG.md]: Can you remove the changelog addition.  I'll handle that when I cut the next release, which will probably be 6.0

File: recipes/default.rb [hunk 2]
```diff
@@ -215,6 +215,15 @@
   mode '0755'
 end
 
+
+# resource.cfg differs on RPM and tarball based systems
+when 'rhel', 'fedora'
+directory node['nagios']['resource_dir'] do
+  owner "root"
```
Comment (tas50) [inline recipes/default.rb]: This should be single quotes

File: CHANGELOG.md (hunk 1 above)
Comment (ghost) [inline CHANGELOG.md]: Not a problem, it's deleted.

File: recipes/default.rb (hunk 2 above)
Comment (ghost) [inline recipes/default.rb]: Not a problem, I've updated it to use siingle quotes.

File: recipes/default.rb [hunk 3]
```diff
@@ -215,6 +215,15 @@
   mode '0755'
 end
 
+
+# resource.cfg differs on RPM and tarball based systems
+when 'rhel', 'fedora'
```
Comment (tas50) [inline recipes/default.rb:220]: I fixed the missing case here in https://github.com/tas50/nagios/commit/0353471e5b9fde801816583a158abd2cabf3020a

**Pull request merged**